keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -75,14 +61,13 @@ class ResourceController(
   }
 
   @PutMapping(
-    path = ["/{name}"],
```

Comment:
Specifically I think `PUT` would be right if the user was specifying the name (i.e. the full URL of the resource's `GET`)

Reply:
👍